components/ResultsPageComponents/CommuteDistanceDistribution/CommuteDistanceDistributionChart.jsx

```diff
@@ -17,27 +19,39 @@ export default function CommuteDistanceDistributionChart({ title, data }) {
   const highChartsOptions = {
     title: {
       text: title,
+      style: { color: "#022640" },
     },
     chart: {
       type: "column",
+      height: 400,
     },
     colors: ["#044B7F"],
     tooltip: {
-      pointFormat: "{point.y}%",
+      pointFormat: "{point.y}",
     },
     xAxis: {
       categories: Object.keys(data),
+      title: {
+        text: "Distance",
+      },
+      labels: {
+        formatter: function () {
+          return this.value + " km";
+        },
+      },
     },
     yAxis: {
       min: 0,
       title: {
-        text: "distance",
+        text: "No. of people",
```

Comment:
Hi @RCode-Blue ,
sorry, after reading this [slack discussion](https://codeforaustralia.slack.com/archives/C03JWTLMQ07/p1660518449013049?thread_ts=1660134365.572529&cid=C03JWTLMQ07) more carefully, I realised the y-axis values IS referring to percentages... 🙇🏽  
```suggestion
        text: "% of staff that travelled to work",
```